hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapIssueTest.java

```diff
@@ -232,8 +232,64 @@ public void testMapPagingValues() {
         final PagingPredicate predicate = new PagingPredicate(pageSize);
         predicate.nextPage();
 
-        final Collection<Integer> values = map.values(predicate);
+        Collection<Integer> values = map.values(predicate);
         assertEquals(pageSize, values.size());
+
```

Comment:
this test seems hard to understand to me. Is it possible to add some small explanatory comments on each code block like `this is done for this` or `now we are doing this` ?